storage/redis/run-test.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+source ../../util.sh
+
+echo "------------------------------------"
+echo "Cleaning up running Docker containers and volumes, and key material..."
+echo "------------------------------------"
+docker-compose down
+docker-compose rm -f -v
+
+echo "------------------------------------"
+echo "Starting Docker containers..."
+echo "------------------------------------"
+docker-compose up -d
+waitForDCService nodeA
+waitForDCService nodeB
+
+echo "------------------------------------"
+echo "Creating root"
+echo "------------------------------------"
+curl -s -X POST http://localhost:11323/internal/vdr/v1/did >/dev/null
+waitForTXCount "NodeA" "http://localhost:11323/status/diagnostics" 1 10
+waitForTXCount "NodeB" "http://localhost:21323/status/diagnostics" 1 10
+
+# create 20 new DID documents on each node
+echo "------------------------------------"
+echo "Creating transactions"
+echo "------------------------------------"
+for _ in {1..20}
+do
+   curl -s -X POST http://localhost:11323/internal/vdr/v1/did >/dev/null
+   curl -s -X POST http://localhost:21323/internal/vdr/v1/did >/dev/null
+done
+
+echo "------------------------------------"
+echo "Performing assertions..."
+echo "------------------------------------"
+waitForTXCount "NodeA" "http://localhost:11323/status/diagnostics" 41 10
+waitForTXCount "NodeB" "http://localhost:21323/status/diagnostics" 41 10
```

Comment:
aren't the number of transactions now tracked by the node itself? If so, this test only verifies that the redis storage doesn't kill the node...

Reply:
In memory you mean, so maybe if we restart before asserting?

Reply:
yes, that would at least retrieve something from storage before asserting

Reply:
TX count is read from the DB when requesting diagnostics

Reply:
I agree that although the assertion in itself is sound, restarting would be an extra assertion, given it's a storage test.